samples/Gonda/Gonda.Client/Models/TokensResponse.cs

```diff
@@ -0,0 +1,19 @@
+using Newtonsoft.Json;
+
+namespace Gonda.Client.Models
+{
+    public class TokensResponse
```

Comment:
Once you move to the IM client, this class shouldn't be necessary.

Reply:
removed ^_^